axiom/monitors_integration_test.go

```diff
@@ -58,14 +58,19 @@ func (s *MonitorsTestSuite) SetupTest() {
 	var err error
 	s.monitor, err = s.client.Monitors.Create(s.ctx, axiom.MonitorCreateRequest{
 		Monitor: axiom.Monitor{
-			AlertOnNoData: false,
-			APLQuery:      fmt.Sprintf("['%s'] | summarize count() by bin_auto(_time)", s.datasetID),
-			Description:   "A test monitor",
-			Interval:      time.Minute,
-			Name:          "Test Monitor",
-			Operator:      axiom.BelowOrEqual,
-			Range:         time.Minute * 5,
-			Threshold:     1,
+			AlertOnNoData:                false,
+			APLQuery:                     fmt.Sprintf("['%s'] | summarize count() by bin_auto(_time)", s.datasetID),
+			Description:                  "A test monitor",
+			Interval:                     time.Minute,
+			Name:                         "Test Monitor",
+			Operator:                     axiom.BelowOrEqual,
+			Range:                        time.Minute * 5,
+			Threshold:                    1,
+			SecondDelay:                  10,
```

Comment:
```suggestion
			SecondDelay:                  time.Second * 10,
```

Reply:
Maybe add an assertion on the created monitor to make sure it is really 10 minutes?

Reply:
ten seconds?